RF24Network.cpp

```diff
@@ -1181,7 +1181,7 @@ template<class radio_t>
 uint64_t ESBNetwork<radio_t>::pipe_address(uint16_t node, uint8_t pipe)
 {
 
-    static uint8_t address_translation[] = {0xc3, 0x3c, 0x33, 0xce, 0x3e, 0xe3, 0xec};
+    static uint8_t address_translation[] = {0xc3, 0x3c, 0x33, 0xce, 0x3e, 0xe3, 0xec, 0xee, 0xed};
```

Comment:
```suggestion
    static uint8_t address_translation[] = {0xc3, 0x3c, 0x33, 0xce, 0x3e, 0xe3, 0xec
#if NUM_PIPES > 6
, 0xee
    #if NUM_PIPES > 7
, 0xed
    #endif
#endif
};
```
I wrote this from my phone, so clang-format will probably "fix" that suggestion.